testing/trino-testing/src/main/java/io/trino/testing/BaseConnectorTest.java

```diff
@@ -2274,6 +2274,7 @@ private List<SetColumnTypeSetup> setColumnTypeSetupData()
                 .add(new SetColumnTypeSetup("array(integer)", "array[1]", "array(bigint)"))
                 .add(new SetColumnTypeSetup("row(x integer)", "row(1)", "row(x bigint)"))
                 .add(new SetColumnTypeSetup("row(x integer)", "row(1)", "row(y integer)", "cast(row(NULL) as row(x integer))")) // rename a field
+                .add(new SetColumnTypeSetup("row(x integer, y integer)", "row(1, 2)", "row(x integer, z integer)", "cast(row(1, NULL) as row(x integer, z integer))")) // rename a field, but not all fields
```

Comment:
What if we swap a variable name in case of row ?

Reply:
there already is x,y -> y,x swapping test case